sale_warn_option/models/product_template.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    sale_line_warn_option = fields.Many2one(
+        comodel_name="warn.option",
+    )
+
+    @api.onchange("sale_line_warn_option")
+    def _onchange_sale_line_warn_option(self):
```

Comment:
Why not compute?

Reply:
I only want this to happen on the Interface to ensure the User sees the new message